notebooks/243-tflite-selfie-segmentation/243-tflite-selfie-segmentation.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    def postprocess_mask(out, pad_info, orig_img_size):
</pre><p>Do we need to add function argument types?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
type hints and docs strings added for all functions